### DISCUSSION TITLE
Cookie method doesn't recognize cookie.

### DISCUSSION THREAD

**Discussion by sugoidesune** [nuxt-community/auth-module#1546] [Support Requests]
### Version
module: 5.0.0-1624817847.21691f1
nuxt: 2.15.7

### Nuxt configuration
 - [ x] universal

### Nuxt configuration

    
```
auth: {
    strategies: {
      cookie: {
        cookie: {
          // (optional) If set we check this cookie exsistence for loggedIn check
          name: 'PHPSESSID',
        },
        endpoints: {
          login: {
            url: '/login',
            method: 'post',
          },
          logout: {
            url: '/logout',
            method: 'get',
          },
          user: {
            url: '/profiles/me',
            property: false,
            // autoFetch: false,
            method: 'get',
          },
        },
      },
    },
  },
```

#### What is expected?
After succesful login, status of $auth changes to `loggedIn=true`

#### What is actually happening?
Status of $auth is `loggedIn=false`

## Additional information
I am getting the correct cookie in the login response.
![image](https://user-images.githubusercontent.com/20317691/131650129-3076af29-182d-42d1-be2e-cffda23994ba.png)

I am even setting the user via `this.$auth.setUser` but in both cases still status is still `loggedIn=false`

The cookie is also not used for fetching `\user`

I don't know where to start debugging this. I can provide more information.

## Checklist
* [ x]  I have tested with the latest Nuxt version and the issue still occurs
* [ ]  I have tested with the latest module version and the issue still occurs
* [ x]  I have searched the issue tracker and this issue hasn't been reported yet

**Comment by jezeniel** [comment]
Potentially related to this issue https://github.com/nuxt-community/auth-module/issues/1256

**Comment by Tragio** [comment]
@sugoidesune is your cookie HTTP Only? If so you can't have the `name: 'PHPSESSID'`, because JS can't read it.

Also, make sure it is Lax and is Secure, some browsers have weird behaviors if that is not set.
